core/src/Cuda/Kokkos_Cuda_Instance.cpp

```diff
@@ -893,6 +893,92 @@ const cudaDeviceProp &Cuda::cuda_device_prop() const {
   return m_space_instance->m_deviceProp;
 }
 
+namespace Impl {
+
+extern int get_gpu(const InitArguments &args);
+
+int g_cuda_space_factory_initialized =
+    initialize_space_factory<CudaSpaceInitializer>("150_Cuda");
```

Comment:
What do all of these numbers in `the initialize_space_factory<*SpaceInitializer>(...)` calls mean? Maybe, it was explained somewhere but I couldn't find it.

Reply:
This is addressed in 494054f. I can copy this comment to all of the uses, or refer back to it at all of the usage sites, or something like that if you think it would be helpful

Reply:
Thanks! No, it's fine with me.